src/Symfony/Component/AstGenerator/Util/AstHelper.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AstGenerator\Util;
+
+use PhpParser\Node\Expr;
+use PhpParser\Node\Scalar;
+
+final class AstHelper
+{
+    /**
+     * Transforms a php value into an AST node.
+     *
+     * @param null|bool|int|float|string|array $value
+     *
+     * @return Expr
+     */
+    public static function value($value)
+    {
```

Comment:
I forget to mention to use var_export, that's what i'm using here : https://github.com/jolicode/jane-openapi/blob/65fe879139b8eaad18d132e850cebadd85f88dca/src/Generator/Parameter/NonBodyParameterGenerator.php#L67